parse/bestprice.js

```diff
@@ -0,0 +1,26 @@
+
+const parseBestPrice = (ctx, outDaySeg, journeys) => {
+	const {profile, res} = ctx;
+
+	const bpjourneys = outDaySeg.conRefL
+		? outDaySeg.conRefL
+			.map(i => journeys.find(j => j.refreshToken == res.outConL[i].ctxRecon))
+			.filter(j => Boolean(j))
+		: [];
+
+	const amount = outDaySeg.bestPrice.amount / 100;
+	const currency = bpjourneys?.[0]?.price?.currency;
+
+	const result = {
+		journeys: bpjourneys,
+		fromDate: profile.parseDateTime(ctx, outDaySeg.fromDate, outDaySeg.fromTime),
```

Comment:
Given that these two fields also contain a time (that is not always `00:00`, according to the test fixture), I propose renaming them to either `{from,to}DateTime` or just `{from,to}`. What do you think?